client/my-sites/upgrades/checkout/domain-details-form.jsx

```diff
@@ -132,7 +158,7 @@ export default React.createClass( {
 			name,
 			ref: name,
 			additionalClasses: 'checkout-field',
-			value: formState.getFieldValue( this.state.form, name ),
+			value: formState.getFieldValue( this.state.form, name ) || '',
```

Comment:
Is this change required by the new field? Maybe it should be fixed there (so that it properly returns an empty string when no value is input), instead of here?